Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Init.hs

```diff
@@ -155,17 +161,23 @@ handler merchantId req eitherReq = do
             Nothing -> logDebug "Unable to get the key"
 
           return (booking, Nothing, Nothing)
-        Left _ -> throwError $ InvalidRequest "Can't have driverQuote in specialZone booking"
+        _ -> throwError $ InvalidRequest "Can't have driverQuote in specialZone booking"
+    InitRentalReq -> do 
+      case initReq of
+        RENTAL_QUOTE (rentalQuote, searchRequest) -> do
+          booking <- buildRentalBooking rentalQuote searchRequest  rentalQuote.id.getId now DRB.RentalBooking now (mbPaymentMethod <&> (.id)) paymentUrl searchRequest.disabilityTag
```

Comment:
why forth parameter `startTime = now?` It should be some time in future, correct?

Reply:
yes